static/src/js/components/Facets/FacetsItem.jsx

```diff
@@ -103,31 +103,44 @@ class FacetsItem extends Component {
             checked={applied}
             onChange={this.onFacetChange.bind(this, changeHandlerArgs)}
           />
-          <span className="facets-item__title">
-            {facet.title}
+          <div className="facets-item__title-container">
             {
-              facet.description
-            && (
-              <OverlayTrigger
-                placement="top"
-                overlay={
-                  (
-                    <Tooltip style={{ width: '20rem' }}>
-                      {facet.description}
-                    </Tooltip>
-                  )
-                }
-              >
+              facet.iconProps && (
```

Comment:
Truthy checks on an object are not a good idea. Destruct `iconProps` off of `facet` before the `return` block and make sure that `icon` exists here before rendering the EDSCIcon component